CHANGELOG.md

```diff
@@ -1,8 +1,3 @@
-# Change Log
```

Comment:
While we're here, can we change it to `# Changelog`? I don't know of any other project that uses two words 😄

Reply:
Ha yeah true